token_benchmark_ray.py

```diff
@@ -90,6 +91,11 @@ def get_token_throughput_latencies(
             prompt_tokens_stddev=stddev_input_tokens,
             expect_output_tokens=num_output_tokens,
         ))
+
+    if log_prompts:
+        print("Sending the following prompts:")
+        print(prompts)
```

Comment:
is it better to print one prompt-per-line? what will be easier to look at

Reply:
didn't you want to log the prompt length in tokens? much more interesting.

I imagine:
per-line
${prompt len in tokens} - $prompt

Reply:
Also, maybe when logging=False just print the prompt lengths (in one line or a histogram of lengths in tokens)

Reply:
Prompts is an array of tuples (prompt, token_count). 

I'll add the log of the tokens